README.md

```diff
@@ -1200,7 +1200,6 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
   * [PmWiki](http://www.pmwiki.org) - Wiki-based system for collaborative creation and maintenance of websites. `GPLv3` `PHP`
   * [Raneto](http://raneto.com/) - Raneto is an open source Knowledgebase platform that uses static Markdown files to power your Knowledgebase. `MIT` `NodeJS`
   * [Realms](http://realms.io/) - A git-backed wiki inspired by Gollum. ([Source Code](https://github.com/scragg0x/realms-wiki)) `GPLv2` `Python`
-  * [Redaxscript](http://redaxscript.com/) - Ultra lightweight website engine. ([Source Code](https://github.com/redaxmedia/redaxscript)) `GPLv3` `PHP`
```

Comment:
Redaxscript is indeed a CMS and should be moved to the appropriate section.
But Redaxo is another project, so there is no duplicate here.

Can you move redaxscript to the CMS section? (instead of removing it)

Thanks